x-pack/elastic-agent/spec/filebeat.yml

```diff
@@ -5,7 +5,8 @@ args: [
   "-E", "setup.template.enabled=false",
   "-E", "management.enabled=true",
   "-E", "logging.level=debug",
-  "-E", "gc_percent=${FILEBEAT_GOGC:100}"
+  "-E", "gc_percent=${FILEBEAT_GOGC:100}",
+  "-E", "metricbeat.config.modules.enabled=false"
```

Comment:
There is an issue about Elastic Agent using a different configuration a `filebeat.elastic-agent.yml`. It would be good to create that file and configure filebeat to run how Filebeat wants Elastic Agent to run it, that way changes can just be done there without having to keep changing the spec file.

There has also been some issues before where these settings are sometimes loaded latter than the configuration from disk not giving the behavior expected.

Reply:
++ on using the separate config file. @ph @jlind23 Do we have this change planned in?

Reply:
@ruflin I don't find any issue related to this. @blakerouse could you paste the link here?

Reply:
Found it: https://github.com/elastic/beats/issues/29860 Initial conversation happened here https://github.com/elastic/beats/pull/29809

Reply:
Seems we want to go #29860, let's close this PR, the other issue is candidate for another release.

Reply:
The problem is there are current issues with the model we have. We had 2 PR's so far we closed because we want to do https://github.com/elastic/beats/issues/29860 which I think is the right path forward. But I think we should prioritise it higher as it might remove some issues.

Reply:
Higher than 8.3 does not seem to be feasible.